code/modules/lore_codex/lore_data/important_locations.dm

```diff
@@ -15,7 +15,7 @@
 		/datum/lore/codex/page/rota
 		)
 
-/datum/lore/codex/page/vir/add_content()
+/datum/lore/codex/page/virgo_erigone/add_content()
 	name = "Vir (Star)"
```

Comment:
This is **not** right.

Reply:
This is lore_data/important_locations.dm that you're looking out (which is commented out. See the .dme) not lore_data_vr/important_locations.dm. 
This file you are commenting on has no say in our current codex book but is right for the new system.

Reply:
so wait then why is it a .dm and not _vr.dm?

Reply:
-/datum/lore/codex/page/vir/add_content()
+/datum/lore/codex/page/virgo_erigone/add_content()

name = "Vir (Star)"

none of this looks right

Reply:
Ace. You are looking at lore_data/important_locations.dm in this comment. Not lore_data_vrr/important_locations.dm

Also, I don't know why it's not a _vr.dm. Ask @Arokha https://github.com/VOREStation/VOREStation/commits/master/code/modules/lore_codex/lore_data_vr/important_locations.dm

Reply:
_Oohhhh_

Reply:
nevermiiiiiind